getting_started/first_3d_game/01.game_setup.rst

```diff
@@ -9,15 +9,15 @@ importing the start assets and setting up the game scene.
 We've prepared a Godot project with the 3D models and sounds we'll use for this
 tutorial, linked in the index page. If you haven't done so yet, you can download
 the archive here: `Squash the Creeps assets
-<https://github.com/godotengine/godot-3d-dodge-the-creeps/releases/tag/1.1.0>`__.
+3d_squash_the_creeps_starter.zip <https://github.com/godotengine/godot-docs-project-starters/releases/download/latest-4.x/3d_squash_the_creeps_starter.zip>`_.
 
 Once you downloaded it, extract the .zip archive on your computer. Open the
 Godot Project Manager and click the *Import* button.
 
 .. image:: img/01.game_setup/01.import_button.webp
 
 In the import popup, enter the full path to the freshly created directory
-``squash_the_creeps_start/``. You can click the *Browse* button on the right to
```

Comment:
I named this `3d_squash_the_creeps_starter` to match one of the patterns in https://github.com/godotengine/godot-docs-project-starters/tree/master/src